tests/sentry/incidents/endpoints/test_organization_alert_rule_anomalies.py

```diff
@@ -68,7 +73,28 @@ def test_simple(self, mock_seer_store_request):
 
         self.login_as(self.user)
 
+        seer_return_value = {
```

Comment:
I don't think this return value is correct based on https://github.com/getsentry/seer/blob/main/src/seer/anomaly_detection/models/external.py#L74-L74